### ISSUE TITLE
Milestone 3: Report feedback

### ISSUE THREAD

**Issue by ksedivyhaley** [UBC-MDS/DSCI_522_group-314#55]
**Reasoning:**

- Good notes giving context for the Sept/Aug 2005 repayment status!
- Explanation on Fig 7 is an improvement, but it would still be better to have a more precise description of your model's feedback than "fairly good". (This may be tricky, we can talk about this in lab.) 
- Have you considered creating features relative to credit limit, as indicated by your peer feedback?
- Peer feedback: "Explains precesion/recall and how it's related to default"
  - You do this in the summary ("Due to the risk associated with customers failing to pay, the model was designed to maximize the recall rate, identifing customers that will default to the greatest extent.") but it would also be good to include it in the conclusions - especially since you note that your best-case model actually has lower recall than the base-case. I'd like to to briefly discuss that observation!

> We can see that some of the features have stronger correlation with the response varibale than others, for example LIMIT_BALANCE and Age.

Do you mean that both age and limit balance have stronger correlations with the response variable, or that limit balance has a stronger correlation than age?

**Writing:**

- You're missing references to values from your report, eg: {{auc}}.
- Some grammatical issues, eg: "It shows that that mean, standard deviation, min, max etc." "It's interesting that **The** medians" "Summary the data used in this study"

**Comment by dataubc** [comment]
I will be doing the report modification

**Comment by dataubc** [comment]
- You're missing references to values from your report, eg: {{auc}}.

-The final report is the html file and not the ipynb file since we could render the variables in Jupyter.
https://github.com/UBC-MDS/DSCI_522_group-314/blob/master/doc/final_report.html

The html file has all the variables automatically rendered.

**Comment by dataubc** [comment]
Other issues were fixed as suggested

**Comment by ksedivyhaley** [comment]
I see, the html was not displaying well on github so I was looking at the notebook.

I am noticing now that the project I cloned from github did not produce an html - when re-trying make, I note the error:

```
ValueError: Unknown exporter "html_embed", did you mean one of: asciidoc, custom, html, latex, markdown, notebook, pdf, python, rst, script, slides?
```

Do you know if this is a dependency issue?

**Comment by ribner** [comment]
@ksedivyhaley , hmm we ran into this error on linux machines before. But we changed the makefile and it seemed to resolve it.
```
#create final report .html	
doc/final_report.html : results
	jupyter nbconvert --execute  doc/final_report.ipynb --to html
```
I see no html_embed in the current makefile, is it possible your local copy is out of sync?

**Comment by dataubc** [comment]
We addressed this issue here. It was only for linux system and fixed it

https://github.com/UBC-MDS/DSCI_522_group-314/issues/51

**Comment by ksedivyhaley** [comment]
>I see no html_embed in the current makefile, is it possible your local copy is out of sync?

I was testing your Milestone 3 version, as I needed to ensure that the Makefile as submitted for the Milestone would result in a properly rendered report. Hadn't noticed/remembered the previously resolved issue with the same system-dependent bug when responding to this issue.

Confirmed that the properly rendered html has the values in the correct place.
